flex/07-flex-layout-2/solution/solution.css

```
@@ -73,11 +73,11 @@ a {
   padding: 32px;
   display: flex;
   flex-wrap: wrap;
+  gap: 16px;
```

```suggestion
  gap: 32px;
```
To match the original desired outcome image, you'd need a gap of 32px as the previous 16px margin would be doubled (no margin collapse).

---

updated gap to 32px.